src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -32,16 +32,14 @@ class AccessDecisionManager implements AccessDecisionManagerInterface
     private $allowIfEqualGrantedDeniedDecisions;
 
     /**
-     * Constructor.
-     *
-     * @param VoterInterface[] $voters                             An array of VoterInterface instances
-     * @param string           $strategy                           The vote strategy
-     * @param bool             $allowIfAllAbstainDecisions         Whether to grant access if all voters abstained or not
-     * @param bool             $allowIfEqualGrantedDeniedDecisions Whether to grant access if result are equals
+     * @param iterable $voters                             An iterator of VoterInterface instances
```

Comment:
`iterable` is equivalent to `array|\Traversable` so we still accept arrays as before.